testcode/CMakeLists.txt

```diff
@@ -78,6 +81,9 @@ add_test(rescaling rescaling)
 add_test(parrescaling mpirun -np 2 ./rescaling)
 add_valgrind_test(vrescaling rescaling)
 
+add_test(coarse_assembling coarse_assembling -m 1 --perm ${SPE10_PERM})
```

Comment:
Thanks for adding tests! (we should also have some kind of `lineargraphthree` test but that will be my responsibility in the parent branch).